openapi.json

```diff
@@ -151,7 +151,7 @@
         ]
       }
     },
-    "/report/{orgId}/{clusterId}": {
+    "/organizations/{orgId}/clusters/{clusterId}/users/{userId}/report": {
```

Comment:
Why do we need userId and organization in the url? AFAIK we should get this data in `x-rh-identity` header.

Reply:
I guess it is related to move the authentication to the smart proxy service. Aggregator won't have authentication anymore, so the smart proxy (or manual testing) can provide the user id in the URL instead of authentication

Reply:
@quarckster because of the new architecture. We don't want to pass extra information such as authentication token to the aggregator, so we pass user_id directly in parameters.

If you want to see endpoint for users, it's here https://github.com/RedHatInsights/insights-results-smart-proxy/pull/39/files#diff-e2493ceba9636a2a265230ffacb30ed8